dvc/repo/experiments/push.py

```diff
@@ -121,7 +121,7 @@ def _notify_studio(
     import requests
     from requests.adapters import HTTPAdapter
 
-    endpoint = urljoin(url or STUDIO_URL, "/webhook/dvc")
+    endpoint = urljoin(url or STUDIO_URL, "/webhook/dvc/")
```

Comment:
I'd prefer Studio change the canonical url to the one without slash. Although merging this for the time being. cc @amritghimire